### ISSUE TITLE
With regard to energy_comparison.dat

### ISSUE THREAD

**Issue by haory-95** [FitSNAP/FitSNAP#260]
Looked through all the folders and couldn't find anything about “xxx.dat”（e.g., I'm going to run plot_energy_comparison.py from the example ）, how can I get these data files, please? Thanks

**Comment by rohskopf** [comment]
See the PyTorch examples.

**Comment by rohskopf** [comment]
Hi @haory-95 , see e.g. Ta PyTorch NN example. These energy comparison files were replaced with perconfig.dat

**Comment by haory-95** [comment]
> Hi @haory-95 , see e.g. Ta PyTorch NN example. These energy comparison files were replaced with perconfig.dat

Thank you very much for your reply, I have found this information with your guidance. But I would also like to ask, what should I do if I want to get other dat information besides the Ta_Pytorch_NN folder (e.g. Fe, but I only find loss_vs_epochs.dat in Fe_Pytorch_NN)

**Comment by rohskopf** [comment]
@haory-95 there is a flag in the Ta example which dumps this data.

```
[EXTRAS]
dump_peratom = 1   # write per-atom fitting metrics
dump_perconfig = 1 # write per-config fitting metrics
```

If you use these flags in those examples, you will get the per atom and per config metrics.

**Comment by haory-95** [comment]
@rohskopf Do you mean to run the “example.in” ile directly, and then the corresponding “.dat" file will be generated automatically after the run is finished?

**Comment by rohskopf** [comment]
Yes but those files are only for nonlinear fits. Linear fits use a dataframe.
